beam-core/Database/Beam/Backend/SQL/AST.hs

```diff
@@ -479,7 +479,7 @@ instance IsSql92TableNameSyntax TableName where
 data TableSource
   = TableNamed TableName
   | TableFromSubSelect Select
-  | TableFromValues [ [ Expression ] ]
+  | TableFromValues Int [ [ Expression ] ]
```

Comment:
Would love to see a little comment here about what this `Int` represents

Reply:
This is the number of columns in the result (in the case that the expressions list is empty, we wouldn't be able to infer that from the values).

Reply:
I meant as documentation, just like you did in `tableFromValues`